src/rules/block-no-empty/__tests__/index.js

```diff
@@ -48,3 +48,17 @@ testRule(rule, {
     column: 18,
   } ],
 })
+
+testRule(rule, {
+  ruleName,
+  config: [undefined],
+  skipBasicChecks: true,
+  syntax: "sugarss",
+
+  reject: [{
+    code: ".a",
+    message: messages.rejected,
+    line: 1,
+    column: 3,
```

Comment:
I actually don't know if this will cause a problem for text editors or not (do they always have one free column at the end of lines where you might add another character?), I'm just worried because we've had trouble with this in the past.